src/sources/splunk_hec/mod.rs

```diff
@@ -44,6 +54,8 @@ pub struct SplunkConfig {
     /// A list of tokens to accept. Omit this to accept any token
     valid_tokens: Option<Vec<String>>,
     tls: Option<TlsConfig>,
+    /// Splunk HEC indexer acknowledgement settings
+    indexer_acknowledgements: HecAcknowledgementsConfig,
```

Comment:
This will, of course, be rewritten when #10150 lands.

Reply:
Updated in be21c3a